internal/pkg/flink/internal/controller/interactive_output_controller.go

```diff
@@ -114,30 +113,33 @@ func (t *InteractiveOutputController) inputHandlerTableView(event *tcell.EventKe
 		}
 		return nil
 	case tcell.KeyEscape:
-		t.exitTViewMode()
-		return nil
+		fallthrough
 	case tcell.KeyCtrlQ:
 		t.exitTViewMode()
 		return nil
 	case tcell.KeyEnter:
-		t.openRowView()
+		t.renderRowView()
 		return nil
+	case tcell.KeyUp:
+		fallthrough
+	case tcell.KeyDown:
+		return t.handleKeyUpOrDownPress(event)
 	}
 	return event
 }
 
 func (t *InteractiveOutputController) getActionForShortcut(shortcut string) func() {
 	switch shortcut {
-	case "Q":
+	case components.ExitTableViewShortcut:
 		return t.exitTViewMode
-	case "M":
+	case components.ToggleTableModeShortcut:
 		return t.renderAfterAction(t.resultFetcher.ToggleTableMode)
-	case "A":
+	case components.ToggleAutoRefreshShortcut:
 		return t.renderAfterAction(t.resultFetcher.ToggleAutoRefresh)
-	case "H":
-		return t.tableView.FastScrollUp
-	case "L":
-		return t.tableView.FastScrollDown
+	case components.JumpUpShortcut:
+		return t.stopAutoRefreshOrScroll(t.tableView.FastScrollUp)
```

Comment:
Nice, but hopefully we won't add more layers to this func stacking we have happening here haha

Reply:
haha yeah, I just want to avoid the case blocks getting too big by just doing only one thing